exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java

```diff
@@ -50,15 +54,33 @@ public final class ZipkinSpanExporter implements SpanExporter {
       BytesEncoder<Span> encoder,
       BytesMessageSender sender,
       Supplier<MeterProvider> meterProviderSupplier,
+      HealthMetricLevel healthMetricLevel,
+      Attributes additonalHealthAttributes,
       OtelToZipkinSpanTransformer transformer) {
     this.builder = builder;
     this.encoder = encoder;
     this.sender = sender;
-    this.exporterMetrics =
-        sender.encoding() == Encoding.JSON
-            ? ExporterMetrics.createHttpJson("zipkin", "span", meterProviderSupplier)
-            : ExporterMetrics.createHttpProtobuf("zipkin", "span", meterProviderSupplier);
     this.transformer = transformer;
+
+    String transportName;
+    String componentType;
+    if (sender.encoding() == Encoding.JSON) {
+      transportName = "http-json";
+      componentType = "zipkin_http_json_exporter";
+    } else {
+      transportName = "http";
+      componentType = "zipkin_http_exporter";
```

Comment:
According to semconv, the fully qualified name should be used when non of the standardized values apply:

> otel.component.type: If none of the standardized values apply, implementations SHOULD use the language-defined name of the type. E.g. for Java the fully qualified classname SHOULD be used in this case.

Oddly tho, the standardized values don't mention zipkin, which seems like an omission: https://github.com/open-telemetry/semantic-conventions/blob/02681893953519164eefd086df9e20c7b15a159d/model/otel/registry.yaml#L64-L133